Bottom: ce40de69ded7d2ed2fd575fb332a3ca853016c2b
Top:    a79a71624a0e34ac5baaba8b5311c5067f757077
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 17:27:54 +0100

Refresh of increase-initial-heap-size-fro

---

diff --git a/src/heap.c b/src/heap.c
index 39dcbe4..e889d7e 100644
--- a/src/heap.c
+++ b/src/heap.c
@@ -165,7 +165,7 @@ heap_insert (heap_t * heap, cost_t cost, void *data)
     {
       heap->max *= 2;
       if (heap->max == 0)
-	heap->max = 32;		/* default initial heap size */
+	heap->max = 512;		/* default initial heap size */
       heap->element =
 	realloc (heap->element, heap->max * sizeof (*heap->element));
     }
